WcaOnRails/app/views/registrations/edit_registrations.html.erb

```diff
@@ -54,44 +44,28 @@
 
               <td class="wca-id">
                 <% if registration.wca_id %>
-                  <%= render "shared/wca_id", wca_id: registration.wca_id %>
+                  <%= link_to registration.wca_id, "/results/p.php?i=#{registration.wca_id}" %>
```

Comment:
I'd prefer having a relative: true parameter to render "shared/wca_id".

Maybe it should be a view helper rather than a partial.